packages/react-router-dev/package.json

```diff
@@ -85,7 +85,7 @@
     "semver": "^7.3.7",
     "set-cookie-parser": "^2.6.0",
     "valibot": "^0.41.0",
-    "vite-node": "^1.6.0"
+    "vite-node": "3.0.0-beta.2"
```

Comment:
This was bumped because v3 is the first major version of vite-node to support both Vite v5 and v6: https://github.com/vitest-dev/vitest/blob/main/packages/vite-node/package.json#L89. I think it's okay we're running a beta version because we're using vite-node in a very limited capacity to run config files and we don't expose any Vite internals to consumers. However, just to be sure, we run the routes.ts tests against both v5 and v6 of Vite.